### ISSUE TITLE
Bug: Radar component fell off of User Profile Page

### ISSUE THREAD

**Issue by sdras** [netlify/explorers#184]

**Comment by tzmanics** [comment]
@sdras & @jlengstorf is this still a feature we want to add?

**Comment by jlengstorf** [comment]
@tzmanics yes, once we have more categories I think this is a great visualization. there's some trickiness in figuring out how to track these and show meaningful data, but there's maybe an argument to make for not worrying about being super meaningful? I don't know. I love the look of it, but I suppose the biggest question is whether we have a strong enough benefit from creating it to justify the work
